### DISCUSSION TITLE
Cannot read properties of undefined (reading 'pop') during Next.js build process"

### DISCUSSION THREAD

**Discussion by ItsPriyadharsani** [adobe/react-spectrum#5849] [Q&A]
>npm run dev

> employee-details@1.0.0 dev
> next dev

 ✓ Ready in 10.7s
 ○ Compiling / ...
 ⨯ ./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[6].oneOf[14].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[6].oneOf[14].use[2]!./node_modules/next/dist/build/webpack/loaders/resolve-url-loader/index.js??ruleSet[1].rules[6].oneOf[14].use[3]!./node_modules/next/dist/compiled/sass-loader/cjs.js??ruleSet[1].rules[6].oneOf[14].use[4]!./src/assets/scss/main.scss
Cannot read properties of undefined (reading 'indexOf')

Import trace for requested module:
./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[6].oneOf[14].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[6].oneOf[14].use[2]!./node_modules/next/dist/build/webpack/loaders/resolve-url-loader/index.js??ruleSet[1].rules[6].oneOf[14].use[3]!./node_modules/next/dist/compiled/sass-loader/cjs.js??ruleSet[1].rules[6].oneOf[14].use[4]!./src/assets/scss/main.scss
./src/assets/scss/main.scss

**Comment by snowystinger** [comment]
Can you provide a reproduction repo or stackblitz? are you using React Spectrum components? React Aria Components? just the hooks?
Did this work before adding whichever library from us? or did you just add sass as well?
Could you try excluding our packages from your sass-loader if you haven't already? though I don't see it trying to import our CSS, so it really looks like it's an error trying to import your own scss file?
